src/transformState.js

```diff
@@ -4,8 +4,31 @@
  * @param {Object} state
  * @param {Object[]} actions
  */
+const ADD_PROPERTIES = 'addProperties';
+const REMOVE_PROPERTIES = 'removeProperties';
+const CLEAR_PROPERTIES = 'clear';
```

Comment:
One more cool variant

```suggestion
const ActionType = {
  AddProperties: 'addProperties',
  RemoveProperties: 'removeProperties',
  Clear: 'clear',
};
```